packages/devtools_app/assets/scripts/inspector_polyfill_script.dart

```diff
@@ -74,6 +74,7 @@ import 'package:flutter/src/widgets/debug.dart';
 import 'package:flutter/src/widgets/framework.dart';
 import 'package:flutter/src/widgets/gesture_detector.dart';
 import 'package:flutter/src/widgets/widget_inspector.dart';
+// ignore: depend_on_referenced_packages
```

Comment:
Why not add a dependency on `package:vector_math` tp the pubspec?

Reply:
Good question! This dart file likely shouldn't participate in the package's direct dependencies? It's not used in the context of this package, but is loaded into a running flutter app and eval'd via the service protocol.

I'm guessing that this should actually be re-worked to:
- remove the ignore
- add a dev_dep on package:vector_math